src/tests/test_filesystem_mounting.py

```diff
@@ -0,0 +1,181 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide
+def test_filesystem_mounting(selenium):
+    import asyncio
+    from string import Template
+
+    from pyodide.code import run_js
+
+    PYODIDE1_FLAG = "pyodide1"
+    PYODIDE2_FLAG = "pyodide2"
+
+    # setup the pyodide objects in js and return them to the python code
+    setup_creation_in_js = Template(
+        """
+        async function set_pyodide_objects() {
+            let pyodide1 = await loadPyodide();
+            let pyodide2 = await loadPyodide();
+
+            await pyodide1.FS.mkdir("${PYODIDE1_DIR}");
+            await pyodide2.FS.mkdir("${PYODIDE2_DIR}");
+
+            await pyodide2.FS.mount(
+                pyodide2.FS.filesystems.PROXYFS,
+                {
+                    root: "${PYODIDE1_DIR}",
+                    fs: pyodide1.FS,
+                },
+                "${PYODIDE2_DIR}"
+            );
+            self.pyodide1 = pyodide1;
+            self.pyodide2 = pyodide2;
```

Comment:
These assignments look redundant

Reply:
They are used later [here](https://github.com/pyodide/pyodide/blob/8e72959cd8771347c91041c609e93524eea642de/src/tests/test_filesystem_mounting.py#L99-L103). That's why they are assigned.

As you [commented](https://github.com/pyodide/pyodide/pull/5830/files#r2264348671), I will modify it and then remove them if possible.